AfricanReligions/events/afr_roog_religious_traditions.txt

```diff
@@ -0,0 +1,121 @@
+################################################
+#
+# Events for Religious Traditions
+#
+#################################################
+
+namespace = GROUPS
+
+long_character_event = { #Landing Event
+	id = GROUPS.1000
+	title = "GROUPSTITLE"
+	desc = "EVTDESCGROUPS1000"
+	picture = "GFX_evt_tribal_shrine"
+
+	is_triggered_only = yes
+
+	trigger = { NOT = { is_inaccessible_trigger = yes } }

```

Comment:
Let's not add a trigger to this event that's only going to be triggered from the decision - it just creates a circumstance where the player clicks on the decision by the event doesn't appear.  If you want this to be a requirement, put it in the `allow` block of the decision.